### ISSUE TITLE
Schreibe Text funktionier nicht immer

### ISSUE THREAD

**Issue by UweLoyal** [ioBroker/ioBroker.text2command#187]
Folgende Einstellung habe ich gemacht:

![image](https://user-images.githubusercontent.com/5464553/209207947-d738bd84-8a74-49de-9fe3-b157db1cefa8.png)

Wenn ich jetzt sage: "Alexa frage ioBroker suche Sender Pro 7"
antwortet er mit: "Folgender Text gesendet: pro 7"

Wenn ich nach welt frage, dann klappt nicht, bei welt 7 klappt wieder alles!

Woran kann dies liegen?

LG
Uwe Loyal

**Comment by Apollon77** [comment]
Du nutzt den Custom Skill?  Dier Standard Variante oder die neue die wir in "Beta" Testen?

Das "Problem" ist das es sehr schweirgig ist "den echten text" abzugreifen ... der neue Custom SKill versucht das anders und könnte dein Problem ggf lösen ... siehe https://forum.iobroker.net/topic/60835/test-custom-skill-v2-f%C3%BCr-iobroker-iot?_=1671736443999

Ist am Ende kein problem von text2command weil ich annehme das bei "Welt 7" dies ggf gar nicht ankommt als Test oder?

**Comment by UweLoyal** [comment]
Doch, es kam wie oben beschrieben "welt 7" in 0_userdata.0.Alexa.TV_Button.SkyQ.Suche_Sender an.
Wenn ich sage "Alexa frage ioBroker suche Sender Super RTL" dann kommt "kommt Ich verstehe '?' nicht".

![image](https://user-images.githubusercontent.com/5464553/209304746-194081e2-6be9-4eb7-9b4d-e30145cf530d.png)

![image](https://user-images.githubusercontent.com/5464553/209305055-6e4a132f-ba7c-49c9-a5d2-0a97fd84d539.png)
iot.0.smart.lastCommandObj 
{"words":["?"],"intent":"controlDevice","deviceId":"amzn1.ask.device.xxxx

LG
Uwe

**Comment by Apollon77** [comment]
Das json ist unvollständig aber ja das bräuchte man als grundlage für jedes - oder halt text2command debug log aktivieren und dann dort das log zeigen

Zweiteres ist vllt das einfacste um zu sehen was rein kamund was als rtespinse rausging

**Comment by UweLoyal** [comment]
![image](https://user-images.githubusercontent.com/5464553/209430767-a863547e-9e1d-48b8-a124-aeab7f1633e6.png)

Alexa frage ioBroker Suche nach Sender Pro 7
![image](https://user-images.githubusercontent.com/5464553/209430729-c8accc8e-6bb1-4e7c-a7b4-96550738ff98.png)
iot:
![image](https://user-images.githubusercontent.com/5464553/209432008-017b1c5c-3ae8-4459-a91a-0756c6f0517a.png)
![image](https://user-images.githubusercontent.com/5464553/209431946-ab6f4742-7882-4e34-adf9-5b6ddcbc14e8.png)


Alexa frage ioBroker Suche nach Sender Welt
![image](https://user-images.githubusercontent.com/5464553/209432558-be26110c-db55-4186-95c7-2f58aefdddeb.png)
iot:
![image](https://user-images.githubusercontent.com/5464553/209432482-37ddf5bd-8830-4e51-815b-706d42737be9.png)

**Comment by Apollon77** [comment]
Ja, dann liegt es am "alten"/aktuellen Custom Skill. Siehe oben und ggf mit neuem Skill testen oder warten bis der neue SKill offiziell kommt.Da könnte/sollte es besser sein
